### PR TITLE
ui enhancement  showing  failures in bulk email task

### PR DESCRIPTION
**Pull request by Abdul-Muqadim-Arbisoft**

### CODE REVIEW

File: lms/djangoapps/instructor_task/views.py [hunk 1]
```diff
@@ -219,14 +221,32 @@ def get_task_completion_info(instructor_task):  # lint-amnesty, pylint: disable=
...
+        detailed_skip_msg = "Detailed Skips: " + ", ".join([f"{reason}: {email}" for email, reason in skip_info])
+
+    if detailed_failure_msg and detailed_skip_msg:
+        detailed_msg = detailed_failure_msg + "; " + detailed_skip_msg
+    elif detailed_failure_msg:
+        detailed_msg = detailed_failure_msg
+    elif detailed_skip_msg:
+        detailed_msg = detailed_skip_msg
+    else:
+        detailed_msg = "No detailed failure or skip information available."
```
Comment (ahmed-arb) [inline lms/djangoapps/instructor_task/views.py]: nit: this is a redundant `else` statement. We can initialize `detailed_msg` with this value.

File: lms/djangoapps/instructor_task/views.py [hunk 2]
```diff
@@ -219,14 +221,32 @@ def get_task_completion_info(instructor_task):  # lint-amnesty, pylint: disable=
...
         succeeded=num_succeeded,
         attempted=num_attempted,
         total=num_total,
         skipped=num_skipped,
         student=student
-    )
-    return (succeeded, message)
+    ) + f" {detailed_msg}"
+
+    return (succeeded, message)
```
Comment (ahmed-arb) [inline lms/djangoapps/instructor_task/views.py]: Git's requesting a trailing new line here.

**Review by ahmed-arb — APPROVED** [review]
LGTM

File: lms/djangoapps/instructor_task/views.py (hunk 2 above)
Comment (Abdul-Muqadim-Arbisoft) [reply lms/djangoapps/instructor_task/views.py]: concern addressed

**Review by ahmed-arb — APPROVED** [review]
Looks good.  Let move this into development environment.

**Review by ahmed-arb — APPROVED** [review]

**Pull request merged**